.github/workflows/pkg-version-bump.yaml

```diff
@@ -25,9 +25,23 @@ jobs:
       contents: write
       issues: write
     steps:
+      - name: Generate release bot app token
+        id: generate_token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.HIROSYSTEMS_PKG_RELEASE_BOT_APP_ID }}
+          private-key: ${{ secrets.HIROSYSTEMS_PKG_RELEASE_BOT_APP_PEM }}
+
+      - name: Get bot user ID
+        id: bot-user-id
```

Comment:
@deantchi I noticed we usually don't specify steps `id`s.
Do we need to?
If so, it should have consistent casing (`-` vs `_`)

Reply:
Right, `id` is not required for this step.  I will remove it.